edx_sga/static/js/src/edx_sga.js

```diff
@@ -18,40 +17,42 @@ function StaffGradedAssignmentXBlock(runtime, element) {
             var content = $(element).find('#sga-content').html(template(state));
             var form = $(element).find("#student-answer-form");
 
-            ClassicEditor.create( document.querySelector( '#student-answer-textarea' ) )
-            .then( editor => {
-                if (state.uploaded) {
-                    editor.setData(state.uploaded.student_answer);
-                }
-            })
-            .catch( error => {
-                console.error( error );
-            });
+            if (form.length) {
+                var uploaded = state.uploaded;
```

Comment:
shouldn't we rename it to `submitted`, uploaded made sense when user was uploaing files